### DISCUSSION TITLE
Adding a link object

### DISCUSSION THREAD

**Discussion by LearnBeFree** [flxzt/rnote#1287] [Ideas]
It would be extremely nice to have an opportunity to add links to the document  (a button added by user which can get them to the specified object in the document). It is hard to explain, but the exact same functionality is implemented in Miro: 
![image](https://github.com/user-attachments/assets/cccfa5b4-1d14-4735-8bf8-cdf73d3d9f2e)
It would enable us to organize larger documents, create table of contents etc.

The alternative could be adding Styles functionality (Title, Heading 1, Heading 2 etc.) and auto-generating table of contents (like text processors do). 

Thank you for the great app!

**Comment by Doublonmousse** [comment]
Related : #980 #904 #294
